app/controllers/comments_controller.rb

```diff
@@ -44,16 +45,21 @@ def handle_create(comment_params)
     comment.id
   end
 
+  # rubocop:disable Metrics/MethodLength
```

Comment:
It makes sense to add this and this is better than making unnecessary helpers for the sake of preventing Rubocop from complaining. I wonder long term if we should just disable "Metrics/MethodLength" in our general config. I don't see this as fundamental. I think PR reviews should help identify whether to reduce the size of a method.

It looks like there are some other places where `# rubocop:disable Metrics/MethodLength` is used (`# rubocop:disable Layout/LineLength` too). 

This can be tackled in another PR :) Let me know what you think!

Reply:
Yaaa, I agree with this. I was staring at this method like.... how... how can I reduce this even more??? lol

10 lines just seems waaaay too short, and you'll end up having a ton of little unnecessary "helper" methods. 

I think adding a PR to adjust the rubocop rules would be good! Maybe either a new max like...35? 40? Or just remove it all together 🤔

Reply:
Good question. Yeah, maybe we could increase the max for now. Do you wanna take that on in another PR? :)

Reply:
Ya sure :D

Reply:
Looks like we can remove these comments with your rubocop config changes merged!